ibft/instance.go

```diff
@@ -236,11 +260,15 @@ func (i *Instance) BumpRound() {
 	i.processChangeRoundQuorumOnce = sync.Once{}
 	i.processPrepareQuorumOnce = sync.Once{}
 	i.processCommitQuorumOnce = sync.Once{}
-	i.State.Round.Set(i.State.Round.Get() + 1)
+	newRound := i.State.Round.Get() + 1
+	i.State.Round.Set(newRound)
+	metricsIBFTRound.WithLabelValues(string(i.State.Lambda.Get())).Set(float64(newRound))
```

Comment:
we can use i.State.Round.Get() here instead

Reply:
calling `i.State.Round.Get()` will acquire the lock again